### PR TITLE
Rename crate on README.md

### PR DESCRIPTION
**Pull request by zypeh**

### CODE REVIEW

**Comment by zypeh** [comment]
Should have the [API documentation](http://rust-doc.s3-website-us-east-1.amazonaws.com/tokio/master/tokio) changed too ?

File: README.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ First, add this to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-tokio = { git = "https://github.com/tokio-rs/tokio" }
+tokio_proto = { git = "https://github.com/tokio-rs/tokio_proto" }
```
Comment (Torwegia) [inline README.md]: I think the line needs to read:
`tokio-proto = { git = "https://github.com/tokio-rs/tokio-proto" }`

**Comment by zypeh** [comment]
Ops, my bad. Didn't notice it, since I am fiddling the `tokio-minihttp` and messed up

**Comment by carllerche** [comment]
Thanks! Merged: 1a7ba2ca4fabf925280330bea878e91c469f7019

**Pull request closed without merging**